src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -30,6 +34,25 @@
 
 namespace Dynamo.Applications.Models
 {
+    public interface IRevitStartConfiguration : IStartConfiguration
+    {
+        ExternalCommandData ExternalCommandData { get; set; }
+    }
+
+    public struct RevitStartConfiguration : IRevitStartConfiguration
+    {
+        public string Context { get; set; }
+        public string DynamoCorePath { get; set; }
+        public IPreferences Preferences { get; set; }
+        public bool StartInTestMode { get; set; }
+        public IUpdateManager UpdateManager { get; set; }
+        public ISchedulerThread SchedulerThread { get; set; }
+        public string GeometryFactoryPath { get; set; }
+        public IAuthProvider AuthProvider { get; set; }
+        public string PackageManagerAddress { get; set; }
+        public ExternalCommandData ExternalCommandData { get; set; }
+    }
+
```

Comment:
Also, if you're going to refactor this, we should probably shorted `xStartConfiguration` to `xStartConfig`.